scripts/newinstall.sh

```diff
@@ -633,7 +633,7 @@ n8l::python_check() {
 				if [[ $pyverok != true ]]; then
 					cat <<-EOF
 
-					Thanks. After you install Python 2.7 or 3.5 and the required modules,
```

Comment:
Could you use variable interpolation similar to the above `2 (>=2.${minver2}) or 3 (>=3.${minver3})`? That would eliminate a set of hard coded version strings.

Reply:
Removed reference to versions completely. This message comes just after we've already reported the version required.

Reply:
:+1: